wye/templates/workshops/workshop_detail.html

```diff
@@ -107,7 +107,7 @@ <h1 class="no-space">Workshop Details</h1>
                         <div class="row">
                             <div class="col-xs-2"><strong>College representative</strong></div>
```

Comment:
I just rebased with master and created PR, IDK how these files are part of this PR